### DISCUSSION TITLE
After closing Lightbox slide, the scroll is all the way up

### DISCUSSION THREAD

**Discussion by roniaxe** [igordanchenko/yet-another-react-lightbox#109] [General]
Hi,
I'm not raising this as a defect, because in your codesandbox it's not happening, so I assume it's something local at my environment.
However, I'm using Remix, and using PhotoAlbum with Lightbox :
```
<>
            <PhotoAlbum
                photos={imagesMapped}
                layout="rows"
                onClick={({ index }) => setIndex(index)}
            />
            <Lightbox
                slides={slides}
                open={index >= 0}
                index={index}
                close={() => setIndex(-1)}
                plugins={[Captions]}
                captions={{
                    descriptionTextAlign: 'center',
                }}
                animation={{
                    swipe: 100,
                    fade: 500,
                }}
            />
        </>
```

Any time I scroll down, and click a photo, it opens up the Lightbox modal, and when I close it, the scroll is all the way up in the beginning of the page, and lose it's position of before opening the modal. 
Do you know what could be the problem ?

**Comment by igordanchenko** [comment]
Unfortunately, I do not have any suggestions as I've never experienced a similar issue. Do you think you can produce a demo for me to take a look at?

**Reply by roniaxe** [reply]
I will definitely try to create a remix demo similar to what I'm having. Thanks for your quick reply Igor!

**Reply by roniaxe** [reply]
@igordanchenko the problem was, that I had css on my html tag:
	-ms-overflow-style: none;
	scrollbar-width: none;
	overflow-y: scroll;

And that seems to interrupt the yarl css styles, I assume, with the yarl__no_scroll,.. I'm not sure exactly how.
I removed those css styles from my html tag, and it seems to works perfectly now.

**Reply by igordanchenko** [reply]
Thank you for the update, I'm glad you were able to figure it out!
